docs/advanced/mapping.md

```diff
@@ -7,47 +7,91 @@ hide:
 
-### How do we do it?
+LED Maps replace and over-ride the older Gap system. **Note:** that if a `ledmap.json` file exists, the `2d-gaps.json` file will be ignored.
+
+
+# How do we do it?
+
+
+## Create config file
 
 Navigate to the edit page for your WLED device by adding `/edit` to its address - for example, https://my-led-device.local/edit
-Use this edit page to create a file called `ledmap.json`.
+Use this edit page to create a file called `ledmap1.json`, where `1` is incremented for each map you load on your controller.
+
+**Note:** If the filename is `ledmap.json`, the config file ***will not load***; the filename must end in a number even if there's only one config file.
+
 
-`ledmap.json` file needs to be a JSON formatted file with the the key being "map" and the value being an array of numbers representing the new order of pixels. The _position_ of values in the array is the "natural" order of LEDs and the value entered is the new position.
+## Formatting
+
+`ledmap1.json` file needs to be a JSON-formatted file with the key being `"map"` and the value being an array of numbers `[0,1,2,3...]` representing the new order of pixels. The _position_ of values in the array is the "natural" order of LEDs, with the number being the electrical position of a given LED on the linear strand.
   
-The ArduinoJSON library is *****extremely***** white-space sensitive.
-If your `ledmap.json` file is not working, check for white-spaces where they should not be. The LED positions are zero-indexed.
+The ArduinoJSON library is *****extremely***** whitespace sensitive.
+If your `ledmap1.json` file is not working, check for white-spaces where they should not be. 
+
+***Note:*** In the examples below, additional whitespace is added for readability; however, the examples may not work when copied and pasted directly due to this whitespace sensitivity.
+
+LED positions are zero-indexed. If you have 20 LEDs, your LED "addresses" will start at 0 and end at 19, but they do not need to be entered in order or completely, as demonstrated below.
+
+
+## Additional configurations
+
+Multiple maps are supported in the latest versions by using `ledmapx.json`, where x is a number. Maps can be selected in your cfg.json  using `{"ledmap":x,...`, where `x` corresponds to the number in the filename, or in the main settings UI below segments, where any customised names will also be used in the selection list, see below.
+
 
-Multiple maps are supported in the latest versions by using `ledmapx.json` where x is a number. Maps can be selected in a preset using `{"ledmap":x,...`.
+## Complicated maps
 
-Use -1 in the map for gaps/blank/null LEDs.
+LEDs can be mapped in the "map" array in any order, including out-of-order, allowing you to map custom and complex shapes using `-1` in the map for gaps/blank/null LEDs. In addition, not all LEDs in a segment need to be mapped in the map; you can leave out extra LEDs if they're not required.
```

Comment:
-1 translates to 65535 but any number above actual LED count will provide similar result - placeholder for skipped LED.

Reply:
for sake of "keep it simple", I think we can still tell users that "-1" is the best way.